src/display/editor/tools.js

```diff
@@ -804,6 +807,7 @@ class AnnotationEditorUIManager {
       rotation: 0,
     };
     this.isShiftKeyDown = false;
+    this.#editorUndoBar = editorUndoBar;
```

Comment:
To support the suggestion above.
```suggestion
    this._editorUndoBar = editorUndoBar;
```